active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+
+By default, the InfoBar will not be visible. Set the IsOpen property to true in the XAML or code behind to display the information.
+
+XAML
+```xml
+<Grid>
+    <!--Other UI content -->
+    <StackPanel x:Name="ContentArea">
+        <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+            Severity="Warning"
+            Title="Error while saving"
+            Message="Your document was unable to be saved.">
+        </controls:InfoBar>
+    </StackPanel>
+    <!--Other UI content -->
+<Grid>
+```
+C#
+```C#
+public MainPage()
+{
+    this.InitializeComponent();
+
+    if(!SaveSuccessful())
+    {
+        UnsuccessfulSaveNotification.IsOpen = true;
+    }
+}
+```
+
+Here is the visual representation of the info bar in the page.
+
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_NoButton.png)
+
+## Using pre-defined severity levels
+The type of the info bar can be set via the Severity property to automatically set a consistent status color, icon, and assistive technology settings dependent on the criticality of the notification.
+
+Mockups in various Severities to be added.
+![Mockup of InfoBar with no content in different Severity colors and icons for light mode](images/Docked_ToolkitSeverity.png)
+
+![Mockup of InfoBar with no content in different Severity colors and icons for dark mode](images/Docked_ToolkitSeverityDark.png)
+
+## Programmatic dismiss in info bar
+An info bar can be dismissed by the user via the close button or programmatically. If the notification is required to be in view until the status is resolved and you would like to remove the ability for the user to dismiss the info bar, you can set the IsUserDismissable property to false.
+By default, the close button will appear as an 'X' and the IsUserDismissable property is therefore set to true.
+
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="NoInternetNotification"
+        Severity="Error"
+        Title="No Internet"
+        Message="Reconnect to continue working."
+        IsUserDismissable="False">
+    </controls:InfoBar>
+</StackPanel>
+```
+![Mockup of an InfoBar in a Critical state with no close button](images/Critical_IconNoClose.png)
+
+## Custom styling: status color and icon
+Outside of the pre-defined severity levels, the Background and IconSource properties can be set to customize the icon and background color. The info bar will retain the assistive technology settings of the severity defined, or default if none was defined.
+
+A custom background color can be set via the standard Background property and will override the color set by Severity. Please keep in mind content readability and accessibility when setting your own color.
```

Comment:
@MikeHillberg, @ranjeshj, @teaP 

Wanted to highlight an implementation distinction here. Currently, in contrast to the guidance here, the background color of the InfoBar can **not** be custom set via the Background property. Should it? Or, because background color is tied to the Severity level, that we recommend users to overwrite the associated Severity level background brush? @teaP mentioned that if we wanted to allow the Background property to override the background color set by the Severity that we could implement a background panel as an in-between for the Severity brushes and Background property.

Thoughts?

Reply:
Should the severity set a base value which gets overwritten by the Background and Foreground chosen by the app devs?

Or does Severity override all?

Is there any contextual or functional value to having a set severity, but have the look of the control set differently by the app?

Reply:
Conceptually, a Severity is always going to be "set" in that even if a developer doesn't explicitly define a Severity it will follow the "informational" design with a pre-defined background color, icon, and accessibility settings. With that, I don't think Severity should override other custom values and should instead be a baseline.

My opinion is that if a developer wants to overwrite the background color independent of Severity they can. We're allowing them to set a custom icon without having them change the definition of the Severity for their app so why should there be a different pattern for the Background color?

Reply:
It's unfortunate that Background is on Control, since it doesn't necessarily make sense on every control. But in this case I'd expect it to override whatever the control would otherwise do, regardless of the information level.

Reply:
No problem, I'll make it work. 👍